draft-hoffman-dns-over-https-latest.mkd

```diff
@@ -175,32 +179,29 @@ the well-known path allows automated discovery of a DNS API Service,
 and also helps contextualize DNS Query requests pushed over an active
 HTTP/2 connection.
 
-Some forms of the request may also include a HTTP query as defined by
-{{RFC3986}}.
+A DNS API Client encodes the DNS query into the HTTP request
+using the HTTP GET or POST methods.
+The messages can use the on-the-wire representation of a DNS message defined in {{dnswire}}
+or the simple JSON format defined in {{I-D.hoffman-simplednsjson}}.
 
-A DNS API Client encodes the DNS query into the HTTP request in one
-of two different methods:
+When using the POST method, the body MUST use a media
+type selected by the DNS API client, and that media type MUST be indicated by the
+request's Content-Type header.
```

Comment:
are you trying to say that the response media type needs to match the request? why?